Encountify/ViewModels/NewLocationViewModel.cs

```diff
@@ -74,8 +74,8 @@ private async void OnSave()
             {
                 Name = Name,
                 Description = Description,
-                CoordX = CoordX,
-                CoordY = CoordY,
+                Longitude = CoordX,
+                Lattitude = CoordY,
```

Comment:
It would also be easier to read here if it would be Latitude = Latitude, not Latitude = CoordY